gammapy/data/pointing.py

```diff
@@ -125,40 +149,40 @@ def __init__(
             self.__dict__.update(self.from_fits_header(meta).__dict__)
             return
 
-        if not isinstance(mode, PointingMode):
-            raise TypeError(f"mode must be an instance of PointingMode, got {mode!r}")
+        if mode is not None:
+            warnings.warn(
+                "Passing mode is deprecated and the argument will be removed in Gammapy 2.0."
```

Comment:
Since 1.1 is already released and we require the deprecation notice to be in at least one Gammapy release before it is removed, we cannot remove it in 1.2.

Reply:
We need to support this in 1.2, and it will be removed since 1.3. I think we planned 4 minor releases between LTS releases, so 2.0 should come after that.

Would it be possible to use `deprecated_renamed_argument` here, eg: https://github.com/gammapy/gammapy/blob/5764a25fcca02815300c45a058f28799f118fe3d/gammapy/utils/deprecation.py#L35

Reply:
It's not really renamed, its removed, but if it is common, I could add a decorator also for that

Reply:
> We need to support this in 1.2, and it will be removed since 1.3. I think we planned 4 minor releases between LTS releases, so 2.0 should come after that.

Ah ok, I thought only two minor between major releases